src/Cake.Issues.EsLint.Tests/Cake.Issues.EsLint.Tests.csproj

```diff
@@ -27,9 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
-    <PackageReference Include="Cake.Core" Version="1.0.0" />
     <PackageReference Include="Cake.Testing" Version="1.0.0" />
-    <PackageReference Include="Cake.Issues" Version="0.9.0" />
```

Comment:
I think this should not be part of this PR (version will be updated via https://github.com/cake-contrib/Cake.Issues.EsLint/pull/156).

Reply:
`Cake.Issues` reference will be removed, not added or updated